test/extended/util/client.go

```diff
@@ -603,7 +622,13 @@ func (c *CLI) AdminBuildClient() buildv1client.Interface {
 }
 
 func (c *CLI) AdminConfigClient() configv1client.Interface {
-	return configv1client.NewForConfigOrDie(c.AdminConfig())
+	if len(c.staticConfigManifestDir) == 0 {
+		return configv1client.NewForConfigOrDie(c.AdminConfig())
+	} else {
+		// get all config objects from the directory
+		// SetupStaticConfigsFromManifests() needs to be invoked first
+		return fakeconfigv1client.NewSimpleClientset(c.configObjects...)
```

Comment:
you need a shim for this client that causes every mutating request to fail.

Reply:
Done